packages/swingset-xsnap-supervisor/scripts/build-bundle.js

```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env node
-import '@endo/init';
+// Adopting legacy until Babel heals from property override mistakes.
+// https://github.com/endojs/endo/issues/1846
```

Comment:
```suggestion
// TODO Adopting legacy until Babel heals from property override mistakes.
// https://github.com/endojs/endo/issues/1846
```

Or, in our practice, is the issue link enough?

Reply:
This change obviated by https://github.com/endojs/endo/pull/1893. Removed the commit.